tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/GradleUI.java

```diff
@@ -16,12 +16,12 @@
 
 import com.liferay.ide.ui.util.UIUtil;
 
-import org.eclipse.buildship.core.CorePlugin;
-import org.eclipse.buildship.core.event.Event;
-import org.eclipse.buildship.core.event.EventListener;
-import org.eclipse.buildship.core.event.ListenerRegistry;
-import org.eclipse.buildship.core.workspace.ProjectCreatedEvent;
-import org.eclipse.buildship.core.workspace.ProjectDeletedEvent;
+import org.eclipse.buildship.core.internal.CorePlugin;
```

Comment:
We need to review the reasons why we are depending on these internal packages and see if there is a way to accomplish this same thing without using internals.